requirements_docs.txt

```
@@ -11,3 +11,4 @@ sphinx-notfound-page>=0.3.0
 sphinx-copybutton
 sphinx-gallery>=0.4.0
 pydata-sphinx-theme
+pyansys_sphinx_theme
```

```suggestion
pyansys-sphinx-theme
```
It seems that it still installs, but it should be dashes instead of underscores.